src/js/components/Tabs/Tabs.js

```diff
@@ -55,7 +55,13 @@ const Tabs = forwardRef(
           setActiveTitle,
         }}
       >
-        {child}
+        {/* possible to have undefined child. in that case, you can't
+        do cloneElement */}
+        {child
+          ? // need active to be passed as prop for custom styled component
+            // that uses props.active
+            React.cloneElement(child, { active: activeIndex === index })
```

Comment:
I don't think we need to `cloneElement`. I think we can just add `<StyledTab active={active} />`. At least, I tried that on the custom theme story with:
```
extend: ({ active, theme }) => css`
      ${active && 'background: #f00;'}
`
```
and it worked fine.

Reply:
In the case of this user, they weren't applying it in the theme but instead were using a styledcomponent: https://codesandbox.io/s/brave-tdd-ussjb?file=/src/component/buttonTab.js

I tried their code locally with passing active to `StyledTab` and wasn't able to see the proper results.

Reply:
Can we add a comment:
// cloneElement is needed for backward compatibility. We should remove that in our next major release.

Reply:
Wouldn't we want to keep it in the future as well if it's allowing the styled component to be styled properly?

Reply:
It's more of a discovery comment for the purpose of cleaning up the `cloneElement` usage, not sure which direction it will play.

Reply:
Sounds good I will add the comment in.